bcipy/signal/model/pca_rda_kde/pca_rda_kde.py

```diff
@@ -10,15 +10,18 @@
 from bcipy.signal.model.pca_rda_kde.dimensionality_reduction import ChannelWisePrincipalComponentAnalysis
 from bcipy.signal.model.pca_rda_kde.pipeline import Pipeline
 from bcipy.signal.exceptions import SignalException
-from bcipy.helpers.task import TrialReshaper
+from bcipy.helpers.stimuli import InquiryReshaper
 
 
 class PcaRdaKdeModel(SignalModel):
-    reshaper = TrialReshaper()
+    reshaper = InquiryReshaper()
 
-    def __init__(self, k_folds: int):
+    def __init__(self, k_folds: int, prior_type="uniform", pca_n_components=0.9):
```

Comment:
Would it make sense to use an enumeration for `prior_type`?

Reply:
I think for now it's probably equally simple to just use string with the `raise ValueError` clause, since this is only used in one place